_faq/index.md

```diff
@@ -7,32 +7,27 @@ permalink: /docs/faq/
 
 # Babelfish: Frequently asked questions
 
-This page contains some of the most frequently asked question related to
-Babelfish- and Microsoft SQL Server- compatibility.
+This page contains some of the most frequently asked questions related to
+Babelfish and Microsoft SQL Server compatibility.
 
 ## Can we use Babelfish for free?
 
-Yes, Babelfish can be used free of charge. It is open source software which is
-provided under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0) and [PostgreSQL license](https://www.postgresql.org/about/licence/).
+Yes, Babelfish can be used free of charge. It is open-source software, provided under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0) and [PostgreSQL license](https://www.postgresql.org/about/licence/).
 
 ## Is Babelfish 100% compatible with Microsoft SQL Server?
 
-The Babelfish community tries to be as compatible as possible but there will
-always be some limitations. If you want to learn more about these limitations,
-check out our [documentation](/docs/usage/limitations-of-babelfish)  which gives a more detailed description.
+The Babelfish community tries to be as compatible as possible but there are some limitations. To learn more about 
+these limitations, check out our [documentation](/docs/usage/limitations-of-babelfish) for a more detailed description.
 
 Babelfish supports TDS protocol versions 7.1 and higher.
 
 ### Which Babelfish versions are available?
 
-Currently we support addons to PostgreSQL 13 or higher. Support for older 
-versions is not planned. However, there is a string commitment to support
-future versions of PostgreSQL as they are released.
+Currently, Babelfish works with PostgreSQL 13 or higher. Support for older PostgreSQL versions is not planned at this time. 
```

Comment:
This is not true. It currently only works with 13.4. Babelfish 1.1 will only work with 13.5.

Reply:
Updated and checked in... 1a4ef5f..e64cc7a  faq_review -> faq_review